frontend/src/components/ViewLogsPage/ViewLogsPage.tsx

```diff
@@ -230,12 +236,28 @@ export default function DataTable() {
     dispatch(getDropoffs());
   }, [dispatch, pickups, dropoffs]);
 
+
+  function handlePickups() {
+    console.log('Pickups')
+    setPickupSelected((prev) => true);
+    setDropoffSelected((prev) => false);
+  }
+
+  function handleDropoffs() {
+    console.log('Dropoffs')
+    setPickupSelected((prev) => false);
+    setDropoffSelected((prev) => true);
+  }
+
+
   const [buttonText, setButtonText] = React.useState('Dropoffs');
   const handleClick = () => {
     setButtonText(buttonText === 'Dropoffs' ? 'Pickups' : 'Dropoffs');
     if (buttonText === 'Dropoffs') {
+      handleDropoffs()
       temp = 1;
```

Comment:
```suggestion
```
I think we can rid of temp and just use `pickupSelected`/`dropoffSelected` instead 

Then in the function `getData` you can check the values of pickup/dropoffSelected instead of temp